gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/infra/query_service/parameters/ParametersDomainServiceImpl.java

```diff
@@ -15,28 +15,38 @@
  */
 package io.gravitee.apim.infra.query_service.parameters;
 
-import io.gravitee.apim.core.parameters.query_service.ParametersQueryService;
+import io.gravitee.apim.core.parameters.domain_service.ParametersDomainService;
+import io.gravitee.repository.management.model.Parameter;
 import io.gravitee.rest.api.model.parameters.Key;
-import io.gravitee.rest.api.model.parameters.ParameterReferenceType;
-import io.gravitee.rest.api.service.ParameterService;
-import io.gravitee.rest.api.service.common.ExecutionContext;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 import java.util.stream.Collectors;
 import lombok.RequiredArgsConstructor;
+import org.springframework.core.env.ConfigurableEnvironment;
 import org.springframework.stereotype.Service;
 
 @RequiredArgsConstructor
 @Service
-public class ParametersQueryServiceImpl implements ParametersQueryService {
+public class ParametersDomainServiceImpl implements ParametersDomainService {
 
-    private final ParameterService parameterService;
+    private final ConfigurableEnvironment environment;
 
     @Override
-    public Map<Key, List<String>> findAll(ExecutionContext executionContext, List<Key> keys, ParameterReferenceType referenceType) {
-        var parameters = this.parameterService.findAll(executionContext, keys, referenceType);
-        if (null != parameters) {
-            return parameters.entrySet().stream().collect(Collectors.toMap(e -> Key.findByKey(e.getKey()), Map.Entry::getValue));
+    public Map<Key, String> getSystemParameters(List<Key> keys) {
+        return keys
+            .stream()
+            .map(this::getSystemParameter)
+            .filter(Objects::nonNull)
+            .collect(Collectors.toMap(parameter -> Key.findByKey(parameter.getKey()), Parameter::getValue));
+    }
+
+    private Parameter getSystemParameter(Key key) {
+        if (environment.containsProperty(key.key()) && key.isOverridable()) {
```

Comment:
`&& key.isOverridable()` do we need this check here ? It seems strange for a get to not return the parameter if it cannot be overwritten